dutchx_scripts/specsNewDaos/dutchx.ganache.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "DutchX IV",
+  "tokenName": "DutchX",
+  "tokenSymbol": "DX",
+  "founders": [
```

Comment:
dutch x does not have founders

Reply:
one founder with 0 tokens and 0 founders

Reply:
set tokens to 0, added a live config

Reply:
https://github.com/daostack/dx-deployment-script/pull/1/commits/4a606d620bab8af0d3923dbc470b75b855f15642